src/Service.js

```diff
@@ -579,10 +586,12 @@ Service_.prototype.saveToken_ = function(token) {
 
 /**
  * Gets the token from the service's property store or cache.
+ * @param {boolean} optSkipMemory Whether to bypass the local memory cache when
+ *     fetching the token (the default is false).
  * @return {Object} The token, or null if no token was found.
  */
-Service_.prototype.getToken = function() {
-  return this.getStorage().getValue(null);
+Service_.prototype.getToken = function(optSkipMemory) {
+  return this.getStorage().getValue(null, optSkipMemory);
```

Comment:
I'm confused as to what this line means:
```js
return this.getStorage().getValue(null, optSkipMemory);
```
Get `null`? Could we define a better interface to produce more readable code?

Reply:
Added a comment to clarify.